<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка программы</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="GeneralTab">
    <property name="toolTip">
     <string>Общие настройки программы</string>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>277</width>
       <height>113</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="GeneralLayout">
      <item>
       <widget class="QCheckBox" name="StartupCheckBox">
        <property name="text">
         <string>Добавить программу в автозагрузку</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="DBlLayout">
        <item>
         <layout class="QHBoxLayout" name="DBAddressLayout">
          <item>
           <widget class="QLabel" name="DBAddressLabel">
            <property name="text">
             <string>Адрес БД:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DBAddressEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DBUsernameLayout">
          <item>
           <widget class="QLabel" name="DBUsernameLabel">
            <property name="text">
             <string>Имя пользователя БД:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DBUsernameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DBPasswordLayout">
          <item>
           <widget class="QLabel" name="DBPasswordLabel">
            <property name="text">
             <string>Пароль пользователя БД:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DBPasswordEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="NotificationTab">
    <property name="toolTip">
     <string>Оповещения о приближении срока окончания лицензии</string>
    </property>
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QGroupBox" name="LicenseNotificationBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Контроль закупок</string>
     </property>
     <widget class="QCheckBox" name="soundNotificationLicenseCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Звуковое оповещение</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeTextLicenseNotificationBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить текст оповещения</string>
      </property>
     </widget>
     <widget class="QPushButton" name="countLicenseBtn">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>50</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Настройка минимального количества лицензий</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>163</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="intervalLicenseLayout">
       <item>
        <widget class="QLabel" name="intervalLicenseLabel">
         <property name="text">
          <string>Интервал (минут):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalLicenseSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="CustomerNotificationBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Контроль лицензий, у которых истекает срок действия</string>
     </property>
     <widget class="QCheckBox" name="soundNotificationCustomerCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Звуковое оповещение</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeTextCustomerNotificationBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить текст оповещения</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>163</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="intervalCustomerLayout">
       <item>
        <widget class="QLabel" name="intervalCustomerLabel">
         <property name="text">
          <string>Интервал (минут):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalCustomerSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>255</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="daysLayout">
       <item>
        <widget class="QLabel" name="daysLabel">
         <property name="text">
          <string>Дней до окончания действия лицензии:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="daysSpinBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="soundPathLayout">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Путь к звуковому файлу:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseSoundBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
